---
layout: paper
title: Reversed graph embedding resolves complex single-cell developmental trajectories 
image: /images/papers/monocle2.png
authors: Xiaojie Qiu, Qi Mao, Ying Tang, Li Wang,	Raghav Chawla, Hannah Pliner, Cole Trapnell
year: 2017
ref: Qiu et al. 2017.
journal: "bioRxiv"
pdf: /pdfs/papers/qiu-monocle2.pdf
doi: 10.1101/110668
#externurl: "http://biorxiv.org/content/biorxiv/early/2017/02/21/110668.full.pdf"
projects: Single-cell trajectory analysis
---

# Abstract

Organizing single cells along a developmental trajectory has emerged as a powerful tool for
understanding how gene regulation governs cell fate decisions. However, learning the structure
of complex single-cell trajectories with two or more branches remains a challenging
computational problem. We present Monocle 2, which uses reversed graph embedding to
reconstruct single-cell trajectories in a fully unsupervised manner. Monocle 2 learns an explicit
principal graph to describe the data, greatly improving the robustness and accuracy of its
trajectories compared to other algorithms. Monocle 2 uncovered a new, alternative cell fate in
what we previously reported to be a linear trajectory for differentiating myoblasts. We also
reconstruct branched trajectories for two studies of blood development, and show that loss of
function mutations in key lineage transcription factors diverts cells to alternative branches on the
a trajectory. Monocle 2 is thus a powerful tool for analyzing cell fate decisions with single-cell
genomics. 