---
layout: paper
title: Chromatin accessibility dynamics of myogenesis at single cell resolution
image: /images/papers/cicero.png
authors: Hannah Pliner, Jonathan Packer, José L. McFaline-Figueroa, Darren A. Cusanovich, Riza Daza, Sanjay Srivatsan, Xiaojie Qiu, Dana Jackson, Anna Minkina, Andrew Adey, Frank J. Steemers, Jay Shendure, Cole Trapnell
year: 2017
ref: Pliner et al. 2017.
journal: "bioRxiv"
pdf: /pdfs/papers/pliner-cicero.pdf
doi: 10.1101/155473
#externurl: "http://biorxiv.org/content/biorxiv/early/2017/06/26/155473.full.pdf"
projects: Single-cell ATAC-Seq, Gene regulation by noncoding DNA
---

# Abstract

Over a million DNA regulatory elements have been cataloged in the human genome, but linking
these elements to the genes that they regulate remains challenging. We introduce Cicero, a
statistical method that connects regulatory elements to target genes using single cell chromatin
accessibility data. We apply Cicero to investigate how thousands of dynamically accessible
elements orchestrate gene regulation in differentiating myoblasts. Groups of co-accessible
regulatory elements linked by Cicero meet criteria of “chromatin hubs”, in that they are
physically proximal, interact with a common set of transcription factors, and undergo
coordinated changes in histone marks that are predictive of gene expression. Pseudotemporal
analysis revealed a subset of elements bound by MYOD in myoblasts that exhibit early opening,
potentially serving as the initial sites of recruitment of chromatin remodeling and histonemodifying
enzymes. The methodological framework described here constitutes a powerful new
approach for elucidating the architecture, grammar and mechanisms of cis-regulation on a
genome-wide basis.
